server/executor/trigger/grpc.go

```diff
@@ -24,17 +22,12 @@ import (
 )
 
 func GRPC() Triggerer {
-	return &grpcTriggerer{
-		traceProvider: traceProvider(),
-	}
-}
-
-type grpcTriggerer struct {
-	traceProvider *sdktrace.TracerProvider
+	return &grpcTriggerer{}
 }
 
-func (te *grpcTriggerer) Trigger(_ context.Context, test model.Test, tid trace.TraceID, sid trace.SpanID) (Response, error) {
+type grpcTriggerer struct{}
 
+func (te *grpcTriggerer) Trigger(_ context.Context, ctx context.Context, test model.Test, tid trace.TraceID, sid trace.SpanID) (Response, error) {
```

Comment:
this looks like the first context should be called `ctx` and the 2nd `triggerSpanCtx` or something like that